addons/bestja_frontend_fpbz/data/images.xml

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate="1">
+        <record id="image_bg_blue" model="ir.attachment">
+            <field name="name">back_enable.jpg</field>
+            <field name="datas_fname">back_enable.jpg</field>
+            <field name="res_model">ir.ui.view</field>
+            <field name="type">url</field>
+            <field name="url">/web/static/src/img/back_enable.jpg</field>
```

Comment:
Does the `/web/static/src/img/back_enable.jpg` exists? I can't find it. What is the purpose of this code?